CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.7.1
+
+* Fix #115 by updating to new protobuf lite on Android.
+* Regenerated dart protobuf code.
```

Comment:
A minor inconsistency: the first sentence is imperative (as in "fix an issue(!)", "duck and cover!"), the second is not (as in "fixed an issue", "died in vain").

In commit messages it is quite a good practice to use imperative: it's shorter and you don't need to choose between many English tenses. Because of the same reasons I like it in change logs as well (especially given that they _can_ be generated from commit messages), but whatever style you choose, let's try to use it consistently.

Reply:
yess will fix it